frontend/src/app/admin/dashboard/dashboard.component.ts

```
@@ -164,6 +217,331 @@ export class DashboardComponent implements OnInit, OnDestroy {
+      const today = new Date();
+      const todayStr = today.toISOString().split('T')[0];
+
+      filtered = filtered.filter((r: any) => {
+        const startDate = new Date(r.start_date);
+        const startDateStr = startDate.toISOString().split('T')[0];
+
+        switch (this.dateRangeFilter) {
+          case 'today':
+            return startDateStr === todayStr;
+          case 'week':
+            const weekAgo = new Date(today);
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)